website/releases/release-1.0.0-beta1.md

```diff
@@ -0,0 +1,121 @@
+same file group with the conflicts resolved automatically by the query reader and the compactor. The new concurrency
+mode is currently available for preview in version 1.0.0-beta only. You can read more about it under
+section [Model C: Multi-writer](/docs/next/concurrency_control#model-c-multi-writer). A complete example with multiple 
+Flink streaming writers is available [here](/docs/next/flink-quick-start-guide#non-blocking-concurrency-control). You
+can follow the [RFC](https://github.com/apache/hudi/blob/master/rfc/rfc-66/rfc-66.md) and
+the [JIRA](https://issues.apache.org/jira/browse/HUDI-6640) for more details.
+
+### Functional Index
+
+A [functional index](https://github.com/apache/hudi/blob/00ece7bce0a4a8d0019721a28049723821e01842/rfc/rfc-63/rfc-63.md)
+is an index on a function of a column. It is a new addition to Hudi's [multi-modal indexing](https://hudi.apache.org/blog/2022/05/17/Introducing-Multi-Modal-Index-for-the-Lakehouse-in-Apache-Hudi)
+subsystem which provides faster access method and also absorbs partitioning as part of the indexing system. Now you can 
+simply create and drop index using SQL syntax as follows:
+
+```sql
+-- Create Index
+CREATE INDEX [IF NOT EXISTS] index_name ON [TABLE] table_name 
+[USING index_type] 
+(column_name1 [OPTIONS(key1=value1, key2=value2, ...)], column_name2 [OPTIONS(key1=value1, key2=value2, ...)], ...) 
+[OPTIONS (key1=value1, key2=value2, ...)]
+
+-- Drop Index
+DROP INDEX [IF EXISTS] index_name ON [TABLE] table_name
+```
+
+- `index_name` is the name of the index to be created or dropped.
+- `table_name` is the name of the table on which the index is created or dropped.
+- `index_type` is the type of the index to be created. Currently, only `files`, `column_stats` and `bloom_filters` is supported.
+- `column_name` is the name of the column on which the index is created.
+- Both index and column on which the index is created can be qualified with some options in the form of key-value pairs.
+
+To see some examples of creating and using a functional index, please checkout the Spark SQL DDL
+docs [here](/docs/next/sql_ddl#create-index). You can follow
+the [RFC](https://github.com/apache/hudi/blob/master/rfc/rfc-63/rfc-63.md) and
+the [JIRA](https://issues.apache.org/jira/browse/HUDI-512) to keep track of ongoing work on this feature.
+
+### API changes
+
+#### Record Merger API
+
+The `HoodieRecordPayload` interface was deprecated in favor of the new `HoodieRecordMerger` interface in version 0.13.0.
+The new interface has been further enhanced to support all kinds of merging operations. In particular, the new interface
+supports partial merge and support custom checks before flushing merged records to disk. Please check
+the [javadoc](https://github.com/apache/hudi/blob/3a1d4fb03b1ab8e3cf27073053a4fab0a56a26d2/hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecordMerger.java)
+of the API for more details.
+
+#### New FileGroup Reader
```

Comment:
Is this meant only for the Parquet Base file format ? If so, should we call that out ?

Reply:
good point, i'll mention that.